test/aid.js

```diff
@@ -0,0 +1,18 @@
+/* eslint-disable import/first */
+process.env.NODE_ENV = 'test';
+import mongoose from 'mongoose';
+import request from 'supertest';
+import app from '../server/app';
+import Aid from '../server/models/aid';
+
+describe('aids', () => {
+    describe('GET /aids', () => {
+        it('it should GET all the Aids', async () => {
+            request(app)
+                .get('/api/v1/aids')
+                .set('Accept', 'application/json')
+                .expect('Content-Type', /json/)
+                .expect(200);
+        });
+    });
```

Comment:
So you have to send back stubbed out data to verify that you endpoint worked as expected. Something along the lines of:
```
it('it should return a list of all the Aid data', async () => {
    const testAidList = [
        {...
          /** 
          aid data that is returned when this endpoint is hit,
          just create fake data
          */
        }, 
        {...},
        {...}
    ]
    request(app)
        .get('/api/v1/aids')
        // then send that test data
        .send(testAidList)
        .set('Accept', 'application/json')
        .expect('Content-Type', /json/)
        .expect(200);
});
```